screens/index.html

```diff
@@ -0,0 +1,465 @@
+    <link rel="stylesheet" href="../assets/css/section3.css">
+    <link rel="stylesheet" href="../assets/css/adblock.css">
+    <link rel="stylesheet" href="../assets/css/footer.css">
+  </head>
+
+  <body>
+   <div class="wrapper">
+    <button class="scroll-arrowup"><i class="fa-solid fa-arrow-up-long"></i></button>
+    <header class="header">
+      
+      <div class="wheader">
+        <div class="arrow">
+          <div class="arrow-left">
+            <i class="fa-solid fa-arrow-left-long"></i>
+          </div>
+          <div class="arrow-right">
+            <i class="fa-solid fa-arrow-right-long"></i>
+          </div>
+        </div>
+        <div class="wheaderTop">
+          <div class="wheaderTop-logo">
+            <img src="/assets/image/Logo.png" alt="">
+            <span>The gem</span>
+          </div>
+          
+          
+          <input type="checkbox"  id="checkTap">
+          <label class="Header__sidebar" for="checkTap">
+          </label>
+
+          <ul class="wheaderTop-btn " >
+            
+            <li>HOME</li>
+            <li class="wheaderTop-btn--border --color--fff">PAGES</li>
+            <li>BLOG</li>
+            <li>PORTFOLIOS</li>
+            <li>SHOP</li>
+            <li>FEATURES</li>
+            <li>ELEMENTS</li>
+            <div class="wheaderTopOption">
+              <li><i class="fa-solid fa-earth-europe"></i></li>
+              <li><i class="fa-solid fa-magnifying-glass"></i></li>
+              <li><i class="fa-solid fa-basket-shopping"></i> <span class="triangle-left"></span> <span id="box">2</span></li>
+            </div>
+            
+          </ul>
+          <label class="Header__tog" for="checkTap">
+            <span class="Header__icon"></span>
+          </label>
+          
+        </div>
+        <!-- <div class="wrap-arrow">
+          
+        </div> -->
+        <div class="wheaderBottom">
+          
+          <div class="wheaderBottom-icon">
+            
+            <i class="fa-solid fa-ellipsis-vertical"></i>
+            <i class="fa-solid fa-bars"></i>
+          </div>
+          <div class="wheaderBottom-title pad320">
+            <span class="wheaderBottom-title--ws">pricing</span>
+            <span class="wheaderBottom-title--wl">tables</span>
+          </div>
+          <div class="wheaderBottom-text pad320">
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea</p>
+          </div>
+          <div class="wheaderBottom-btn pad320">
+            <button>purchase now</button>
+            <button>more details</button> 
+            
+          </div>
+
+          <div class="slide">
+            <span class="round active-slide"> </span>
+            <span class="round"> </span>
+            <span class="round"> </span>
```

Comment:
nên đặt tên class có ý nghĩa và chính xác hơn và không đặt tên class theo style. Ví dụ 3 chấm tròn mang ý nghĩa là vị trí hiện tại của slide đang trình chiếu => đặt tên class là slide là sai. tương tự em sửa luôn phần đặt tên class là round ở phần project/client ở footer em nhé
```suggestion
          <div class="pagination">
            <span class="dot active-dot"> </span>
            <span class="dot"> </span>
            <span class="dot"> </span>
```